### PR TITLE
community-fixes

### PR DESCRIPTION
**Pull request by aevesdocker**
This PR fixes https://github.com/docker/docs/issues/16018 and https://github.com/docker/docs/pull/14694

### CODE REVIEW

File: desktop/install/linux-install.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To install Docker Desktop successfully, your Linux host must meet the following
 - KVM virtualization support. Follow the [KVM virtualization support instructions](#kvm-virtualization-support) to check if the KVM kernel modules are enabled and how to provide access to the kvm device.
 - **QEMU must be version 5.2 or newer**. We recommend upgrading to the latest version.
 - systemd init system.
-- Gnome or KDE Desktop environment.
+- Gnome or KDE, or MATE Desktop environment.
```
Comment (dvdksn) [inline desktop/install/linux-install.md:24]: Oxford!

```suggestion
- Gnome, KDE, or MATE Desktop environment.
```

**Review by dvdksn — COMMENTED** [review]
it also looks to me like we have two identical (?) screenshots here of localhost:8000:

- compose/images/quick-hello-world-3.png
- compose/images/quick-hello-world-4.png

**Comment by aevesdocker** [comment]
Slight difference in the screenshots, once says 'Hello world' one says 'Hello from Docker'

**Review by dvdksn — APPROVED** [review]
such approve

**Pull request merged**